yt/data_objects/tests/test_firefly.py

```diff
@@ -3,16 +3,36 @@
 from yt.testing import fake_particle_ds, requires_module
 
 
-@requires_module("firefly_api")
-def test_firefly_JSON_object():
+@requires_module("Firefly")
```

Comment:
These tests require `Firefly` to run, but they won't run on the CI unless you add Firefly to the requirements in `setup.cfg`. I think adding Firefly as a dependency in 
```
[options.extras_require]
…
full =
    …
    firefly-vis>=MINIMAL_VERSION
```

Reply:
this is already part of the PR @cphyc

Reply:
It is indeed. For some reason it didn't show up in my review? Or maybe I was stupid? We'll never know!